scripts/items/daybreak.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- ID: 22040
+-- Daybreak
+-- Able to cast "Dispelga"
+-----------------------------------
+---@type TItem
+local itemObject = {}
+
+itemObject.onItemCheck = function(target, item, param, caster)
+end
+
+itemObject.onItemEquip = function(target, item)
+    local mainWeapon = target:getEquippedItem(xi.slot.MAIN)
+    if mainWeapon and mainWeapon:getID() == xi.item.DAYBREAK then
```

Comment:
can simplify to this since you only need the itemid

```suggestion
    if target:getEquipID(xi.slot.MAIN) == xi.item.DAYBREAK then
```